info/welcome.md

```diff
@@ -4,34 +4,51 @@
   # ברוכות הבאות למסטודון!
 
 
-דברים שכדאי לדעת (מנוסח בלשון אשה אבל פונה לכל המגדרים):
+דברים שכדאי לדעת (מנוסח בלשון אשה על פי רוב, אבל פונה לכל המגדרים):
   
 
 - [מסטודון](https://joinmastodon.org/) היא רשת חברתית מבוזרת, שתווותים הוא רק מופע (שרת) אחד מתוכה. מסטודון בעצמה היא חלק מפרוייקט גדול עוד יותר, של רשתות מבוזרות שיכולות לדבר אחת עם השניה, שנקרא לפעמים "הפדרציה" ולפעמים "פדיברס". פרטים נוספים:
   
   &#x2067;  - [Welcome to the Fediverse](https://joinfediverse.wiki/Main_Page/Fancy) (ויקי בכמה שפות)  
   &#x2067;  - [fediverse.info](https://fediverse.info/) (אתר יותר פשוט להבנה)  
-  &#x2067;  - [fediverse.party](https://fediverse.party/)  
-  &#x2067;  - [fediverse.observer](https://fediverse.observer/)  
+  &#x2067;  - [Fedi.Tips](https://fedi.tips/)עוד אתר יותר פשוט להבנה)
+  
```

Comment:
Why the extra line?

Reply:
mistake, I guess...
can you see it aligned to the right ?

Reply:
Also, if you approved, why is the pull request still open ?

Reply:
approve is a code review, not a merge :)

Reply:
As for how it shows up, I am not sure at what URL I am supposed to see that.

however https://tooot.im/terms it shoved RTL. maybe we should add a translation too.

Reply:
interesting...

I meant that here, [that is the link that the About page show]( https://github.com/Toootim/Resources/blob/master/info/welcome.md ), I see LTR:

![LTR on Firefox 106 on Windows](https://user-images.githubusercontent.com/33288162/198384625-2ec39b5d-d961-49e8-bc69-be15f3d75317.png)

On the 
[Tooot.im About Page](https://tooot.im/about/more) page everything's fine, i.e RTL except the layout obviously:

![Tooot.im About Page showing RTL text](https://user-images.githubusercontent.com/33288162/198385157-df6bae1b-463b-4b43-9f61-f77ce70193c5.png)

[Terms](https://tooot.im/terms) actually show nice !

![Tooot.im About Page showing LTR text](https://user-images.githubusercontent.com/33288162/198385494-a3c722b6-8f02-44c2-84e9-35956cd0cf21.png)

Reply:
![image](https://user-images.githubusercontent.com/409600/198386031-8a1bac34-2ec2-4384-9092-0c472daea024.png)
very odd.

about the RTL rendering of Markdown on GH I am less worried about, I doubt they have ever tested RTL on that code.

Reply:
I'm just a bit disappointed. I wanted the thing to look as professional as possible. Maybe a different platform, or inject it straight into toootim via:

![image](https://user-images.githubusercontent.com/33288162/198387199-4dc3d606-4946-4ca4-9897-28db705fbc12.png)

The problem is I don't know HTML at all (I only started yesterday, when I tried to make these changes), so it may take me some time to figure out how to do this without overburdening the page (e.g. using drop-down sections and the like) AND solving all LTR-RTL issues at the same time...

@yair know of any easy tricks from Markdown to RTL-optimized HTML ?

Maybe I should try [This thing](https://github.com/mamal72/rtl-md), then add some tags in HTML...